promql/promqltest/testdata/native_histograms.test

```diff
@@ -754,18 +754,40 @@ eval instant at 10m histogram_mul_div*float_series_0
 eval instant at 10m float_series_0*histogram_mul_div
     {} {{schema:0 count:0 sum:0 z_bucket:0 z_bucket_w:0.001 buckets:[0 0 0] n_buckets:[0 0 0]}}
 
-# TODO: (NeerajGartia21) remove all the histogram buckets in case of division with zero. See: https://github.com/prometheus/prometheus/issues/13934
 eval instant at 10m histogram_mul_div/0
-    {} {{schema:0 count:Inf sum:Inf z_bucket:Inf z_bucket_w:0.001 buckets:[Inf Inf Inf] n_buckets:[Inf Inf Inf]}}
+    {} {{schema:0 count:Inf sum:Inf z_bucket_w:0.001}}
 
 eval instant at 10m histogram_mul_div/float_series_0
-    {} {{schema:0 count:Inf sum:Inf z_bucket:Inf z_bucket_w:0.001 buckets:[Inf Inf Inf] n_buckets:[Inf Inf Inf]}}
+    {} {{schema:0 count:Inf sum:Inf z_bucket_w:0.001}}
 
 eval instant at 10m histogram_mul_div*0/0
-    {} {{schema:0 count:NaN sum:NaN z_bucket:NaN z_bucket_w:0.001 buckets:[NaN NaN NaN] n_buckets:[NaN NaN NaN]}}
+    {} {{schema:0 count:NaN sum:NaN z_bucket_w:0.001}}
+
+eval instant at 10m histogram_mul_div*histogram_mul_div
```

Comment:
Wasn't the idea to introduce `eval_info`? It should be easy to add, see `promql/promqltest/test.go` and how `eval_warn` is handled.

Reply:
Yes, I'll introduce the `eval_info` in a separate PR.

Reply:
Created a PR #14989 for this.

Reply:
Which is merged now, so this can be reworked now.